src/java/voldemort/utils/MetadataVersionStoreUtils.java

```diff
@@ -144,6 +153,7 @@ private static String getPropertiesString(Properties props) {
     public static void setProperties(SystemStoreClient<String, String> versionStore,
```

Comment:
I haven't pulled down your code so I don't have my IDE to analyze usage, but are we still using this API at all? It looks like might have removed all of its previous usages. If that's the case, I encourage you to get rid of it completely.